doc/user/content/release-notes.md

```diff
@@ -54,19 +54,40 @@ Wrap your release notes at the 80 character mark.
 
 {{% version-header v0.5.1 %}}
 
-- Default to using a worker thread count equal to half of the machine's
-  physical cores if [`--workers`](/cli/#worker-threads) is not specified.
-- [`TAIL`](/sql/tail) now sends rows instead of being implemented as a
-  `COPY TO` operation. It has gained `timestamp` and `diff` columns.
+- Add the [`COPY TO`](/sql/copy-to) statement, which sends the results of
+  the statement it wraps via the special PostgreSQL [`COPY` protocol][pg-copy].
+
+- Send the rows returned by the [`TAIL`](/sql/tail) statement to the client
+  normally (i.e., as if the rows were returned by a [`SELECT`](/sql/select)
+  statement) rather than via the PostgreSQL [`COPY` protocol][pg-copy]. The new
+  format additionally moves the timestamp and diff information to dedicated
+  `timestamp` and `diff` columns at the beginning of each row.
   **Backwards-incompatible change.**
```

Comment:
I always appreciate when tools that I'm using put backwards-compatibility notes as the first thing in the release notes, in their own section, should we do that here?

Reply:
Done.